tests/integration/nmstatectl_test.py

```diff
@@ -73,5 +88,12 @@ def test_show_command_with_yaml_format():
     assert LOOPBACK_YAML_CONFIG in out
 
 
+def test_set_command_with_yaml_format():
+    ret = libcmd.exec_cmd(SET_CMD, stdin=ETH1_YAML_CONFIG)
+    rc, out, err = ret
+
+    assert_rc(rc, RC_SUCCESS, ret)
+
+
 def assert_rc(actual, expected, return_tuple):
```

Comment:
IMHO it is strange that the same value is passed twice to the method, once as actual and once in return_tuple.

Reply:
This function does not know that the `expected` is in the `return_tuple`. I do not see anything nicer here, expect perhaps to use a separate function that always creates the string on input at the caller.

Anyway, it is not part of this PR.

Reply:
You are right, it is not part of this PR. However, I do not follow what you are writing. The method is called `assert_rc` and it assumes that the return code is the first value of `return_tuple`. At the same time `actual` implies that it is the return code from `return_tuple`. I guess the method needs a doc string to explain when `actual =! return_tuple[0]` because it does not make sense to me to call it this way.

Reply:
I will try to name and structure it better in a following patch.

Reply:
Thank you!